dev.md

```diff
@@ -2,7 +2,7 @@
 
 ### For Developers
 
-1. Install VirtualBox and Vagrant on a computer with a relatively large memory size.
+1. Install [VirtualBox](https://www.virtualbox.org/) and [Vagrant](https://www.vagrantup.com/) on a computer with a relatively large memory size.
```

Comment:
Can we list the minimal memory and CPU cores ?

Reply:
I added a recommendation config